.github/workflows/lint.yml

```diff
@@ -0,0 +1,34 @@
+name: Lint
+on:
+  push: { branches: [ "master" ] }
+  pull_request: { branches: [ "master" ] }
+
+concurrency:
+  group: lint-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  python-linter:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+        with: { submodules: recursive }
+      - uses: conda-incubator/setup-miniconda@v2
+        with: { mamba-version: "*", channels: "conda-forge", channel-priority: true }
+      - name: install dependencies
+        shell: bash -l {0}
+        run: |
+          mamba env update --quiet -n test -f environment.yml
+          conda list
+      - name: pylint
+        shell: bash -l {0}
+        run: |
+          pylint echemdb
```

Comment:
I am pretty sure that this will fail once it actually runs.

Reply:
Indeed